### PR TITLE
Fix AvBaseInput not setting state properly for managed components

### PR DESCRIPTION
**Pull request by IndifferentDisdain**
AvBaseInput doesn't set this.state.value if value is passed into constructor props, so initial values won't display.

### CODE REVIEW

**Comment by TheSharpieOne** [comment]
You used `initialValue`, but the [`defaultValue`](https://github.com/Availity/availity-reactstrap-validation/blob/master/src/AvBaseInput.js#L107-L117) does the [same thing](https://github.com/Availity/availity-reactstrap-validation/blob/master/src/AvBaseInput.js#L55) already.

**Comment by IndifferentDisdain** [comment]
Gotcha, didn't see that bit. Will fix and re-submit.

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,5 @@ node_modules
 /dist
 /build
 /lib
-/.tmp
\ No newline at end of file
+/.tmp
+/.idea
```
Comment (TheSharpieOne) [inline .gitignore]: This should be in your global git ignore. It's specific to your setup and not this project.

**Comment by TheSharpieOne** [comment]
Closing since the only change is in the .gitignore and I am not accepting that change for the reason commented above.

**Pull request closed without merging**